### PR TITLE
Basic robots.txt with sitemap url

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -4,10 +4,14 @@ theme = "startwords"
...
 enableEmoji = true
 googleAnalytics = "UA-87887700-8"
+enableRobotsTXT = true
+enableGitInfo = true
 
-[taxonomies]
-  author = "authors"
-  tag = "tags"
+# disable taxonomies until we are ready to use them and have templates
+disableKinds = ["taxonomy", "term", "tag"]
```
Comment (thatbudakguy) [inline config.toml:11]: nice, maybe hugo's builds won't emit warnings about this now.

File: config.toml [hunk 2]
```diff
@@ -4,10 +4,14 @@ theme = "startwords"
 copyright = "This work is licensed under a Creative Commons Attribution 4.0 International License."
 enableEmoji = true
 googleAnalytics = "UA-87887700-8"
+enableRobotsTXT = true
+enableGitInfo = true
```
Comment (thatbudakguy) [inline config.toml:8]: what's the `GitInfo` do?

**Review by thatbudakguy — APPROVED** [review]
looks good

File: config.toml (hunk 2 above)
Comment (rlskoeser) [reply config.toml:8]: It uses git commit info for generating last modified dates for content — see https://gohugo.io/variables/git/

Comment (rlskoeser) [reply config.toml:8]: Should we add a comment in the config? 

From what I read this makes the build a little slower, but seems worth it to me. The github actions build is based on a git checkout, so I think it should work.

Comment (thatbudakguy) [reply config.toml:8]: cool! seems worth it to me too. a comment wouldn't hurt.

**Pull request merged**